index.html

```diff
@@ -0,0 +1,319 @@
+              href="https://github.com/Spayco/katchy/"
+              class="card-link link"
+              target="_blank"
+              rel="noreferrer noopener"
+              >See project</a
+            >
+          </div>
+        </div>
+        
+      </div>
+      <div class="about-me">
+        <section class="about-me-section">
+          <h2>About me</h2>
+          <p class="about-me-text">
+            I am a skilled Full-Stack Web Developer who loves making pages with
+            beautiful UI. I have studied at Microverse, and I am experienced in
+            remote work.
+            <br />
+            I've also been a good helping hand to a lot of developers in
+            different communities!
+            <br />
+            if you'd like to get in contact about potential job opportunities,
+            please reach out!
+          </p>
+          <a
+            href="https://docs.google.com/document/d/1lYq0Kb50TujzJzDWGobIchRI17JMnAiZMF9OBQgNU68/edit?usp=sharing"
+            class="link"
+            target="_blank"
+            rel="noreferrer noopener"
+          >
+            Get my resume
+          </a>
+          <img class="about-me-icon" src="./media/svg/icon-2.svg" alt="Icon" />
+        </section>
+        <hr />
+        <div class="about-me-section stacks">
+          <ul>
+            <li class="languages list-title">
+              Languages
+              <ul>
+                <li>JavaScript</li>
+                <li>TypeScript</li>
+                <li>Ruby</li>
+                <li>HTML</li>
+                <li>CSS</li>
+                <li>Sass</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="frameworks list-title">
+              Frameworks
+              <ul>
+                <li>React & Redux</li>
+                <li>Material-UI</li>
+                <li>Jest</li>
+                <li>Bootstrap</li>
+                <li>Ruby on Rails</li>
+                <li>RSPec</li>
+              </ul>
+            </li>
+          </ul>
+          <ul>
+            <li class="skills list-title">
+              Skills
+              <ul>
+                <li>GitHub</li>
+                <li>CodePen</li>
+                <li>GitLab</li>
+                <li>Terminal</li>
+                <li>WSL</li>
+                <li>Remote Pair-Programming</li>
+                <li>Teamwork</li>
+                <li>Mentoring</li>
+                <li>Code Reviews</li>
+                <li>Mobile/Responsive Development</li>
+              </ul>
+            </li>
+          </ul>
+        </div>
+      </div>
+      <section class="contact_form">
+<div>I'm always interested inhearing about new projects,so if you'd like to chat please get in</div>
+<form action="https://formspree.io/f/xpzbrnpq" method="post">
+  <input type="email" placeholder="Full name"  maxlength="10" name = "Full name"  />
+  <input type="text" placeholder="Email address" name = "email" />
```

Comment:
kindly make sure of the right `type` for both name and email `<input>` tags.
And `maxlength` should be **30** instead of **10** to be like the requirements.
![image](https://user-images.githubusercontent.com/68796232/157491961-faa308b2-b874-4d97-b915-b5589a3148cd.png)
![image](https://user-images.githubusercontent.com/68796232/157492050-76bea65a-c43f-47ff-ac48-f79dd8d94510.png)